### PR TITLE
frontend: mui v4 to v5 migration

### PR DESCRIPTION
**Pull request by miguelmota**
Updates frontend to use mui v5. 

Consists of dependency upgrades, updated imports, and theme config updates for mui v5. Should be pretty straightforward.

### CODE REVIEW

File: packages/frontend/package.json [hunk 1]
```diff
@@ -24,14 +24,19 @@
     "not op_mini all"
   ],
   "dependencies": {
+    "@emotion/react": "^11.5.0",
```
Comment (shanefontaine) [inline packages/frontend/package.json]: Pin deps here (unless this is intentional)

FYI I [updated](https://github.com/hop-protocol/hop/blob/develop/.npmrc#L5-L6) `.nvmrc` yesterday to pin by default. I _think_ I like this decision, but not 100% sold on it.

**Review by shanefontaine — APPROVED** [review]
@miguelmota Looks good to me. Does bumping those deps allow us to remove any of the overrides in root `package.json` or `.nvmrc`? I think no because they're related to `react-app-rewind`.

File: packages/frontend/package.json (hunk 1 above)
Comment (miguelmota) [reply packages/frontend/package.json]: Resolved with https://github.com/hop-protocol/hop/pull/547/commits/c82d01c381b94dd4eeac8a1338e5bf575a45ec0d

**Comment by miguelmota** [comment]
@shanefontaine no since they are unrelated to mui, and related to react-app-rewired react-scripts

**Pull request merged**